### PR TITLE
fix conflict with master

### PR DESCRIPTION
**Pull request by smilesun**

### CODE REVIEW

**Review by schoersch — CHANGES_REQUESTED** [review]
@smilesun I added some comments about what I noted. I tried to focus on files related to the parts I work on.

File: domainlab/utils/generate_benchmark_plots.py [hunk 1]
```diff
@@ -19,40 +32,77 @@ def gen_benchmark_plots(agg_results: str, output_dir: str):
 
     agg_results: path to the csv file
     output_dir: path to a folder which shall contain the results
+    skip_gen: Skips the actual plotting, used to speed up testing.
```
Comment (Car-la-F) [inline domainlab/utils/generate_benchmark_plots.py:35]: I don't see skip_gen as an argument to this function, I also don't remember that I have add something like this. @smilesun did you to add this feature?

**Comment by smilesun** [comment]
I did not, I think George did, but I am not sure if we need to merge the
branch of Debbi-mnist or not. Since I saw there is an open PR to merge into
master, that is why I try to fix its conflicts to master. If not needed we
can close the PR

File: domainlab/utils/generate_benchmark_plots.py (hunk 1 above)
Comment (schoersch) [reply domainlab/utils/generate_benchmark_plots.py:35]: Yes, this was me.
I still think we can close this pr

**Pull request closed without merging**